### PR TITLE
Create static oaf pin

### LINKED ISSUE #838: OAF Pin
Create pin for operational aftershock forecasts. This is static content (for now). Attribution should honor product source. Future enhancements may become product-specific.

See attached image for mockup of desired design. A blue background (instead of grey) would also be acceptable.

screen shot 2018-03-22 at 12 28 13 pm

### PR DESCRIPTION
**Pull request by ehunter-usgs**
fixes #838

### CODE REVIEW

File: src/app/executive/oaf-pin/oaf-pin.component.ts [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+@Component({
+  selector: 'executive-oaf-pin',
+  templateUrl: './oaf-pin.component.html',
+  styleUrls: ['./oaf-pin.component.scss']
+})
+export class OafPinComponent {
+
+  public link = '../oaf';
+  @Input() product: any;
+  public title = 'Aftershock Forcast';
```
Comment (jmfee-usgs) [inline src/app/executive/oaf-pin/oaf-pin.component.ts]: Forecast has an `e`

File: src/app/executive/oaf-pin/oaf-pin.component.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+@Component({
+  selector: 'executive-oaf-pin',
+  templateUrl: './oaf-pin.component.html',
+  styleUrls: ['./oaf-pin.component.scss']
+})
+export class OafPinComponent {
+
+  public link = '../oaf';
+  @Input() product: any;
```
Comment (jmfee-usgs) [inline src/app/executive/oaf-pin/oaf-pin.component.ts]: Place the `@Input()` on its own line

File: src/app/executive/executive.module.ts [hunk 3]
```diff
@@ -39,7 +40,8 @@ import { ShakemapPinComponent } from './shakemap-pin/shakemap-pin.component';
     OriginPinComponent,
     PagerPinComponent,
     RegionInfoPinComponent,
-    ShakemapPinComponent
+    ShakemapPinComponent,
+    OafPinComponent
```
Comment (jmfee-usgs) [inline src/app/executive/executive.module.ts]: alphabetize the components, and imports

**Comment by ehunter-usgs** [comment]
replaced by https://github.com/usgs/earthquake-eventpages/pull/1002

**Pull request closed without merging**